docs/technical_reference/unit_models/pressure_exchanger.rst

```diff
@@ -32,6 +32,13 @@ Where the system is also subject to following constraints:
     
     Figure 1. Schematic representation of an energy recovery system using a pressure exchanger.
 
+When setting the ``has_leakage`` configuration option to ``True``, there is 1 additional variables ``leakage_vol`` that must be fixed.
+
+When setting the ``has_mixing`` configuration option to ``True``, there is 1 additional variables ``mixing_vol`` that must be fixed.
+
+When setting the ``pressure_exchange_calculation`` configuration option to ``PressureExchangeType.high_pressure_difference``,
+then there is no ``efficiency_pressure_exchanger`` variable, users need to fix ``high_pressure_difference`` instead.
```

Comment:
Don't forget to update this, once you allow pressure exchanger efficiency to be created even with high_pressure_difference option. See Adam's comment